charts/amoro/templates/amoro-configmap.yaml

```diff
@@ -0,0 +1,128 @@
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    {{- include "amoro.labels" . | nindent 4 }}
+data:
+  ## Helm chart provided Amoro configurations
+  config.yaml: |
+    ams:
+      admin-username: admin
+      admin-password: admin
+      server-bind-host: "0.0.0.0"
+      server-expose-host: "127.0.0.1"
+
+      thrift-server:
+        max-message-size: 104857600 # 100MB
+        selector-thread-count: 2
+        selector-queue-size: 4
+        table-service:
+          bind-port: {{ .Values.server.table.port }}
+          worker-thread-count: 20
+        optimizing-service:
+          bind-port: {{ .Values.server.optimizing.port }}
+
+      http-server:
+        bind-port: {{ .Values.server.rest.port }}
+
+      refresh-external-catalogs:
+        interval: 180000 # 3min
+
+      refresh-tables:
+        thread-count: 10
+        interval: 60000 # 1min
+
+      self-optimizing:
+        commit-thread-count: 10
+
+      optimizer:
+        heart-beat-timeout: 60000 # 1min
+        task-ack-timeout: 30000 # 30s
+
+      blocker:
+        timeout: 60000 # 1min
+
+      # optional features
+      expire-snapshots:
+        enabled: true
+        thread-count: 10
+
+      clean-orphan-files:
+        enabled: true
+        thread-count: 10
+
+      sync-hive-tables:
+        enabled: true
+        thread-count: 10
+
+      database:
+        type: derby
+        jdbc-driver-class: org.apache.derby.jdbc.EmbeddedDriver
+        url: jdbc:derby:/tmp/amoro/derby;create=true
+
+      #  MySQL database configuration.
+      #  database:
+      #    type: mysql
+      #    jdbc-driver-class: com.mysql.cj.jdbc.Driver
+      #    url: jdbc:mysql://127.0.0.1:3306/db?useUnicode=true&characterEncoding=UTF8&autoReconnect=true&useAffectedRows=true&useSSL=false
+      #    username: root
+      #    password: root
+
+      terminal:
+        backend: local
+        local.spark.sql.iceberg.handle-timestamp-without-timezone: false
+
+    #  Kyuubi terminal backend configuration.
+    #  terminal:
+    #    backend: kyuubi
+    #    kyuubi.jdbc.url: jdbc:hive2://127.0.0.1:10009/
+
+
+    #  High availability configuration.
+    #  ha:
+    #    enabled: true
+    #    cluster-name: default
+    #    zookeeper-address: 127.0.0.1:2181,127.0.0.1:2182,127.0.0.1:2183
```

Comment:
High availability is recommended in production environments, so it is recommended to open this part of the configuration to users.